src/Microsoft.Diagnostics.Monitoring.HostingStartup/ParameterCapturing/ParameterCapturingStrings.resx

```diff
@@ -117,6 +117,9 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="DeniedMethodsFormatString" xml:space="preserve">
+    <value>The following methods are not allowed to have their parameters captured: {0}</value>
```

Comment:
This comment refers to the methods that are not allowed, but the usage of this string is in terms of the method descriptions. Consider better describing that the method descriptions are not allowed.